### PR TITLE
Add guide for Jira gitlab action

### PR DESCRIPTION
**Pull request by phalbert**
# Description

- An action that uses GitLab pipelines to create a Jira ticket
- Fix broken link in similar action for GitHub

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Report Bug | Jira GitLab action (`docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md`)

## Updated docs pages

- Report Bug | Jira GitHub action(`/create-self-service-experiences/setup-backend/github-workflow/examples/Jira/report-a-bug.md`)

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+    - job: create-jira-issue
+      artifacts: true
+  except:
+    - pushes
+  before_script:
+    - apk update
+    - apk add --upgrade curl jq -q
+  script:
+    - |
+      echo "Creating Port entity to match new S3 bucket"
```
Comment (stavbernazport) [inline docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md:222]: Please revise and match the text to be relevant :)

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+      ISSUE_SUMMARY=$(cat $TRIGGER_PAYLOAD | jq -r '.payload.properties.short_title')
+      ISSUE_DESCRIPTION=$(cat $TRIGGER_PAYLOAD | jq -r '.payload.properties.description')
+      ISSUE_TYPE=$(cat $TRIGGER_PAYLOAD | jq -r '.payload.properties.issue_type')
+      JIRA_PROJECT_KEY=$(cat $TRIGGER_PAYLOAD | jq -r '.payload.properties.project')
+
+      curl -X POST \
+          -H 'Content-Type: application/json' \
+          -H "Authorization: Bearer $accessToken" \
+          -d '{"statusLabel": "Creating Entity", "message":"🚀 Creating the instance in Port!"}' \
```
Comment (stavbernazport) [inline docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md:240]: Same here

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md (hunk 1 above)
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md:222]: thank you. I have resolved this

File: docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md (hunk 2 above)
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/gitlab-pipeline/examples/report-a-bug.md:240]: alright

**Review by stavbernazport — APPROVED** [review]

**Pull request merged**